manifests/telegraf/agent.pp

```diff
@@ -104,9 +104,16 @@
     notify => Service['telegraf'],
   }
 
+  # Only the latest telegraf package seems to be available for Ubuntu
+  $version_ensure = $facts['os']['name'] ? {
```

Comment:
Thanks for taking a look.  Looking at [telegraf::install](https://github.com/voxpupuli/puppet-telegraf/blob/1dd7bb230b8dbe1a880f19bdeedb34fd3f42e238/manifests/install.pp#L80-L81), there doesn't seem to be a way to install the package other than the repo.  If they only provide the latest package, aren't we limited to using `latest` on Ubuntu?  Should we put a note in the README about this limitation?

Reply:
That may work. It just seems odd to override a parameter that a user could set when we are looking to set a default. It could be possible that a user has a different repo set up or influx changes the way they do packing in the future.

Reply:
Yeah, it is an odd situation.  My inclination is to:

* Add a note about the Ubuntu limitation to the README
* Add a warning if the user has configured `version` to something other than `latest` on Ubuntu.

You think that will work?  We can always change it if the packaging changes.

Reply:
That would work. With that are you setting the default for `latest` on `ubuntu`? Should we just switch to `latest` for every OS?

Reply:
Hmm, we could do that.  My only concern would be if a `2.x` Telegraf package comes out that has breaking changes, or something like that.  Of course that would happen on Ubuntu with this patch.